Modelica/Units.mo

```diff
@@ -1635,8 +1635,8 @@ Real direction[3](unit=\"1\") = to_unit1(v);   // Automatically vectorized call
 
     function to_kWh "Convert from Joule to kilo Watt hour"
       extends Modelica.Units.Icons.Conversion;
-      input SI.Energy J "Joule value";
-      output Modelica.Units.NonSI.Energy_kWh kWh "kWh value";
+      input SI.Energy J "Value in Joule";
```

Comment:
Why is "Joule" in upper case, but in line 1624 in lower case?

Reply:
> Why is "Joule" in upper case, but in line 1624 in lower case?

If replaced "Joule" by "joule" and "Watt" by "watt"